frontend/src/pages/projects/create.tsx

```diff
@@ -0,0 +1,25 @@
+import type { NextPage } from "next";
+import useTranslation from "next-translate/useTranslation";
+import Head from "next/head";
+import styles from "../../styles/Home.module.css";
+import { CreateProjectForm } from "../../components/createProjectForm";
+import { createProjectSubmitHandler } from "../../handlers/createProjectSubmitHandler";
+
+const CreateProject: NextPage = () => {
+    const { t } = useTranslation("common");
+    return (
+        <div className={styles.container}>
```

Comment:
I don't know if this is fixed or not, but i know there was an issue where the navbar didn't expand all the way from left to right. This was fixed by just using `<>` or `<div>` I think Thomas said we couldn't use className ...
Please take a look here #324